trimmer_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'base_decorator'
+class TrimmerDecorator < Decorator
+  def correct_nam
+    @nameable.correct_name.length > 10 ? @nameable.correct_nam[0..9] : @nameable.correct_name
```

Comment:
* Kindly fix the TrimerDecorator class as it doesn't work as expected. HINT 💭 your correct_name method is spelled wrongly as shown in my screenshot below kindly fix it as when i did i got the expected result 
![image](https://user-images.githubusercontent.com/52497246/192597865-d4c86b23-fac2-4e94-86cd-5b1378a40ede.png)

Reply:
@Davidosky007!
Thank you for reviewing my code. I will change it.